test/bigtest/tests/job-executions/job-item-test.js

```diff
@@ -22,7 +21,7 @@ describe('Job item', () => {
     });
 
     it('has correct date label', () => {
-      expect(jobsPane.previewJobs.jobItems(0).dateLabelText).to.be.equal(translation.endedRunning);
+      expect(jobsPane.previewJobs.jobItems(0).dateLabelText).contains('Ended');
```

Comment:
It looks like the developers will have to spend additional resources to support these values in case of some changes in translations. What do you think about it?

Reply:
It is okay in that case I think. This translation won't change in the near future.